src/NuGet.Core/NuGet.CommandLine.XPlat/ExitCodes.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal static class ExitCodes
```

Comment:
I'm not sure if we really need this given the finite, well-known set of values. I couldn't find anywhere in System.Commandline docs where they use constants (https://learn.microsoft.com/en-us/dotnet/standard/commandline/model-binding#set-exit-codes)

Reply:
It is because there is three exit codes(0, 1, 2) we are using here. They all serve different purposes and allow communication between Nuget.CommandLineXPlat and dotnet sdk. It would be using magic numbers (https://github.com/NuGet/NuGet.Client/pull/5638#discussion_r1502202212) if we start returning 0, 1, 2 without any explanation.

Reply:
I'm not sure they're "magic numbers", as they are standard exit codes and they don't change like a MAX_HTTP_RETRY or timeout or something would. Regardless of what they're called, if it helps write cleaner tests or something, that's fine.